AY2021S2-CS2103-T14-1/tp
# Fix #362: Standardize error messages

## What this does
Fixes #362 
Standardizes with #330 

## How to test
1. Launch SunRez
2. Try the following commands and see if the error messages are standardized in format with other INDEX based commands

* `history -1`
* `history 0`
* `history 10000`
